_posts/utah-broadband-outreach-center-releases-revamped-availability-map.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+status: publish
+published: true
+title: Utah Broadband Outreach Center Releases Revamped Availability Map
+author:
+  display_name: Jessie Pechmann
+  login: jpechmann
+  email: jpechmann@utah.gov
+  url: ''
+author_login: jpechmann
+author_email: jpechmann@utah.gov
+date: '2016-02-01 13:31:17 -0700'
+date_gmt: '2016-02-01 20:31:17 -0700'
+categories:
+- Featured
+tags:
+- broadband
+- goed
+- internet
+---
+
+<a href="{{ "images/utah-residential-broadband-map.png" | prepend: site.baseurl }}"><img src="{{ "/images/utah-residential-broadband-map.png" | prepend: site.baseurl }}" alt="" title="BroadbandMap" width="410" height="300" class="inline-text-right" />The <a href="http://broadband.utah.gov/">Utah Broadband Outreach Center</a> in the Governor’s Office of Economic Development (GOED) released an updated residential broadband map today (<a href="http://broadband.utah.gov/map/">broadband.utah.gov/map</a>), providing an enhanced tool to help customers locate available internet services in their communities.
```

Comment:
`images/utah` needs to always start with a `/`. I will fix this but remember in the future.